liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/AbstractLibertySupport.java

```diff
@@ -31,31 +33,29 @@
     /**
      * Maven Project
      * 
-     * @parameter expression="${project}"
-     * @required
-     * @readonly
+     * @Parameter( property="project", required=true, readonly=true )
```

Comment:
In general, it looks confusing if you also have the annotation in the comment block.   If it is not needed in the comment block, I suggest to remove it.